packages/compass-web/src/entrypoint.tsx

```diff
@@ -287,54 +288,56 @@ const CompassWeb = ({
           {...LINK_PROPS}
         >
           <PreferencesProvider value={preferencesAccess.current}>
-            <LoggerAndTelemetryProvider value={loggerAndTelemetry}>
-              <WithAtlasProviders>
-                <AtlasCloudConnectionStorageProvider
-                  orgId={orgId}
-                  projectId={projectId}
-                  autoConnectConnectionId={autoConnectConnectionId}
-                >
-                  <ConnectionsManagerProvider
-                    value={connectionsManager.current}
+            <LoggerProvider value={logger}>
+              <TelemetryProvider value={{ createTrack: () => logger.track }}>
```

Comment:
(leaving as a separate comment just not to mix two issues into one) This should also be a ref so that the value is not changed on every re-render (that would cause every place that accesses the context to re-render too)